Formula/glibc.rb

```diff
@@ -152,12 +153,60 @@ def install
 
     # Delete bootstrap binaries after build is finished.
     rm_rf bootstrap_dir
+
+    # Add ld.so.conf (will be linked to HOMEBREW_PREFIX/etc/ld.so.conf)
+    (prefix/"etc/ld.so.conf").atomic_write <<~EOS
+      # This file is generated by Homebrew. Do not modify.
+      #{lib}  # ensure Homebrew Glibc always comes first
+      include #{etc}/ld.so.conf.d/*.conf
+    EOS
+    chmod "u=rw,go-wx", prefix/"etc/ld.so.conf"
+
+    # Create ld.so.conf.d directories
+    (prefix/"etc/ld.so.conf.d").mkpath
+    chmod_R "go-w", prefix/"etc/ld.so.conf.d"
+
+    # Add README in etc/ld.so.conf.d
+    (prefix/"etc/ld.so.conf.d/README").atomic_write <<~EOS
+      This is the Homebrew's ld configuration directory
+
+      .conf files in this directory will be loaded automatically by ldconfig.
+
+      Files will be included in lexicographical order, so you can control the order of files with a
+      prefix, e.g.:
+
+          00-first.conf
+          50-middle.conf
+          99-last.conf
+    EOS
+    chmod "u=rw,go-wx", prefix/"etc/ld.so.conf.d/README"
+
+    # Add Homebrew lib to ld search paths
+    (prefix/"etc/ld.so.conf.d/98-homebrew.conf").atomic_write <<~EOS
+      #{HOMEBREW_PREFIX}/lib
+    EOS
+    chmod "u=rw,go-wx", prefix/"etc/ld.so.conf.d/98-homebrew.conf"
+
+    # Add system ld search paths (disabled by default)
+    (prefix/"etc/ld.so.conf.d/99-system-ld.conf.keepme").atomic_write <<~EOS
+      # The system ld search paths
+      # If you want Homebrew's ld.so to search for libraries in the system paths, remove the ".keepme"
+      # suffix of this file.
+      # Mixing the Homebrew and system library search paths is very risky and can cause problems. Please
+      # do this only if you know what you are doing, i.e., at your own risk.
+      include /etc/ld.so.conf
+    EOS
+    chmod "u=rw,go-wx", prefix/"etc/ld.so.conf.d/99-system-ld.conf.keepme"
```

Comment:
```suggestion
    (prefix/"etc/ld.so.conf.d/99-system-ld.conf.example").atomic_write <<~EOS
      # The system ld search paths
      # If you want Homebrew's ld.so to search for libraries in the system paths, remove the ".keepme"
      # suffix of this file.
      # Mixing the Homebrew and system library search paths is very risky and can cause problems. Please
      # do this only if you know what you are doing, i.e., at your own risk.
      include /etc/ld.so.conf
    EOS
    chmod "u=rw,go-wx", prefix/"etc/ld.so.conf.d/99-system-ld.conf.example"
```
`99-system-ld.conf*` is referenced twice here and another time in `caveats` -- may be worth extracting into a method.